quickwit-indexing/src/actors/packager.rs

```diff
@@ -489,6 +494,8 @@ mod tests {
         assert_eq!(
             &split.tags.iter().map(|s| s.as_str()).collect::<Vec<&str>>(),
             &[
+                "tag_bool!",
+                "tag_bool:\u{0}\u{0}\u{0}\u{0}\u{0}\u{0}\u{0}\u{1}",
```

Comment:
that does not look great. We can probably do better consider i64 and f64 do better.